website/docs/modules/create.html.markdown

```diff
@@ -118,8 +118,12 @@ standard structure.
 
 * **README**. The root module and any nested modules should have README
   files. This file should be named `README` or `README.md`. The latter will
-  be treated as markdown. The README doesn't need to document inputs or
-  outputs of the module because tooling will automatically generate this.
+  be treated as markdown. There should be a description of the module and 
+  what it should be used for, as well as a basic usage example.  If possible,
+  a visual diagram of the modules should be included. The README doesn't need 
+  to document inputs or outputs of the module because tooling will automatically
+  generate this. All links to other documentation or image resources must be 
```

Comment:
That external contributor made a fair point about that. I will try to fix README rendering to support github-hosted relative links.

Reply:
After some effort and consideration it's not a simple problem. While we _could_ fix for Github, we don't want to commit to reverse-engineering the URL structure of every VCS provider that Public or Private TF registries might ever support.

So this recommendation should remain. I'd maybe add a sentence like: "If you are linking to file or embedding an image contained in the repository itself, use a commit-specific absolute URL so the link won't point to the wrong version of a resource in the future.